packages/pancake-uikit/src/__tests__/components/buttonmenu.test.tsx

```diff
@@ -135,7 +135,7 @@ it("renders correctly", () => {
 
     .c3 {
       background-color: transparent;
-      color: #1FC7D4;
+      color: #8f80ba;
```

Comment:
Unrelated snapshot fix. Must have gone in before the CI started running tests on PRs.